### PR TITLE
Add PPL

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by isuruf** [comment]
@ocefpaf, ready for review

File: recipes/ppl/build.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+chmod +x configure
+
+./configure --prefix=$PREFIX \
+            --with-gmp-include="$PREFIX/include" \
+            --with-gmp-lib="$PREFIX/lib" \
+            --enable-coefficients=mpz \
+            --enable-interfaces=c,c++
+make
+# Following timeouts on CI
```
Comment (ocefpaf) [inline recipes/ppl/build.sh]: You make this reviewer life so easy :smile:

**Comment by ocefpaf** [comment]
> @ocefpaf, ready for review

@isuruf it seems that someone is cancelling the Travis-CI builds on `master` to let the feedstocking process accumulate into one build. that is bad because it is delaying the creation of many feedstocks :unamused:

**Comment by isuruf** [comment]
Yes, the `ntl` repo is waiting for more than a day.
Btw, we should look into https://github.com/conda-forge/staged-recipes/issues/916

**Comment by ocefpaf** [comment]
> Btw, we should look into #916

It would not be that bad if the jobs were not cancelled.

**Pull request merged**